configs/project.go

```diff
@@ -20,6 +20,11 @@ func (c *Configs) MigrateLocalProjectConfig() error {
 		return err
 	}
 
+	// Avoid deleting ~/.railway
+	if projectDir == os.Getenv("HOME") {
```

Comment:
I think this might be an issue on Windows computers? It might be worth checking if `HOME` is empty too. We have issues with Windows support in the past -> https://railway.canny.io/admin/feedback/feature-requests/p/powershell-support?boards=feature-requests

Reply:
Ah hm, good point. That being said, I don't have a Windows box to test on :pensive: